docs/changelog.txt

```diff
@@ -42,10 +42,13 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## Misc Improvements
 
 ## Documentation
+- ``Lua API.rst``:  added ``autoDFAnnouncement``, ``pauseRecenter``, ``recenterViewscreen``
 
 ## API
+- add functions reverse-engineered from announcement code: ``Gui::parseReportString``, ``Gui::autoDFAnnouncement``, ``Gui::pauseRecenter``, ``Gui::recenterViewscreen``
```

Comment:
I probably should have chimed in on your last PR, but I think the only changelog entry we need is this one. Documentation in particular is implied - we don't need to mention in the changelog that a new function was added _and_ that it was documented at the same time. I'm not as picky about the "Lua" section, but historically I have left it out (i.e. I've only considered listing additions to the API to be needed in the "API" section).